conf/ldap_conf.scr

```diff
@@ -27,7 +27,7 @@
 .ldap_conf
 
 `ag_ini(
-  `IniAgent( "/etc/openldap/ldap.conf" ,
+  `IniAgent( "/etc/ldap.conf" ,
```

Comment:
The two `ldap.conf` files have vastly different responsibilities.
In the context of yast, it was never correct to use `openldap/ldap.conf` to begin with, because yast manipulates OS settings rather than application settings. 
Existing changes that are already carried out by the user will stay in `openldap/ldap.conf`. It is not necessary to migrate any content from one file to another.